### DISCUSSION TITLE
关于动态分组浏览

### DISCUSSION THREAD

**Discussion by dragondusts** [the1812/Bilibili-Evolved#4885] [功能建议 / Ideas]
参考 #1279， #1586， #3068， #3809， #3953
以及https://www.bilibili.com/read/cv6744092/

过去的脚本https://greasyfork.org/scripts/396032  已经失效很久，但毕竟需求还在，跪求出手，感谢。

**Comment by the1812** [comment]
#3068 已经拒绝过了
